lms/templates/edxnotes/edxnotes.html

```diff
@@ -112,7 +111,7 @@ <h2 class="placeholder-title">${_('You have not made any notes in this course ye
     <%static:require_module module_name="js/edxnotes/views/page_factory" class_name="NotesPageFactory">
         NotesPageFactory({
             disabledTabs: ${disabled_tabs},
-            notes: ${notes},
+            notes: ${dump_js_escaped_json(notes, NoteJSONEncoder) | n},
```

Comment:
@robrap Can you give a look to only this change? We missed the js escaping for first page of notes that was serialized into the notes tab html. Does it looks ok to you ? Thanks